content/blog/synadia-adaptive-edge.md

```diff
@@ -0,0 +1,80 @@
+
+We see users deploy NATS that fit into a few buckets - a single cluster [k8s](http://kubernetes.io) deployment, a cluster of NATS servers in the cloud on VMs or on bare metal.  As companies grow we see many multi-regional deployments spread out geographically, in data centers, across cloud providers or more often now with hybrid deployments.
+
+More recently we're seeing a pattern emerge with a central group of applications servicing and receiving data from edge nodes.  Usually there is telemetry coming from the edge, and sometimes edge nodes have their own services around command and control and access to local data.  This isn't unusual - it's the federated edge compute with IoT pattern.
+
+What is really interesting is that we're seeing this pattern applied to many different market verticals as businesses push computing to the edge.  Often, users have glued disparate technologies together with different security domains invariably resulting in systems that are fragile, unsecure, and expensive to maintain.  This problem can be elegantly avoided with what we have called the **NATS Adaptive Edge Architecture** - a flexible deployment topology overlayed with NATS multi-tenant security model.
+
+# Security
+
+In NATS 2.0, we enhanced security around the concept of [operators and accounts](https://docs.nats.io/nats-tools/nsc/nsc#creating-an-operator-account-and-user).  An operator is a "owner" of a NATS deployment, such as an Ericsson, Mastercard, or Synadia.  Operators create accounts - think of accounts like *"containers for messaging"* - true multi-tenancy.  An account might contain users representing a group of applications, a regional deployment, or a business unit.  Note that we're moving toward zero trust, so in operator mode, even with this notion of accounts and users, the NATS system **never** stores or accesses a private NATS key.
+
+When a NATS client connects its credentials indicates it belongs to a particular account.  Its subject namespace (where it can send and receive data) exists only within its account.  This means that by default, data will never traverse an account boundary and a client can only directly communicate with other clients in the same account, even if using the same subjects.
+
+However, accounts can import and export streams (think telemetry) or services (think RPC) with other accounts allowing one to securely share specific data and map subjects, effectively decoupling data access from the application subject namespace.  Within a deployment, streams and services may be made public for all accounts to import, or private to  adhere to the strictest of security policies.  Because security is truly bifurcated from connectivity, if one chooses accounts may only exist on a subset of servers to create data silos.
```

Comment:
How about adding emphasis to streams & services?

```suggestion
However, accounts can import and export **streams** (think telemetry) or **services** (think RPC) with other accounts allowing one to securely share specific data and map subjects, effectively decoupling data access from the application subject namespace.  Within a deployment, streams and services may be made public for all accounts to import, or private to  adhere to the strictest of security policies.  Because security is truly bifurcated from connectivity, if one chooses accounts may only exist on a subset of servers to create data silos.
```

Reply:
I added this manually with some other changes we're making trying to get tables to work in hugo.  Thanks!